index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap"
+      rel="stylesheet"
+    />
+    <title>Movie Card</title>
+  </head>
+
+  <body>
+    <section class="js-card-hover card">
+      <img
+        src="https://cdn.glitch.com/a90fa73f-cda0-467b-8b6b-14e862d382eb%2FCard%20One.jpg?v=1603071730629"
+        alt=""
```

Comment:
Without an `alt` value here, a screen reader user would get no information from your image. At the very least, it should include the movie title.